pkg/config/config.go

```diff
@@ -95,8 +97,13 @@ func GetConfig(cfgFilename string) (*Config, error) {
 	if err != nil {
 		return nil, fmt.Errorf("fatal error config file: %s", err)
 	}
-	// Convert to second.
-	config.MetricsInterval = config.MetricsInterval * time.Second
+
+	if config.MetricsIntervalDuration != 0 {
+		config.MetricsInterval = config.MetricsIntervalDuration
+	} else {
```

Comment:
I guess this is bug-for-bug compatible, but setting neither field would generate the deprecation warning unless I missed a presence check somewhere up the call stack.

Reply:
No, I missed it completely, Will fix it thanks!